cmd/ocm/login/cmd.go

```diff
@@ -265,6 +274,16 @@ func run(cmd *cobra.Command, argv []string) error {
 		cfg.User = ""
 		cfg.Password = ""
 	}
+
+	// If an OCM region is provided, update the config URL with the SDK generated URL
+	if cfg.RhRegion != "" {
+		regValue, err := rhRegion.GetRhRegion(connection.URL(), args.rhRegion)
+		if err != nil {
+			return fmt.Errorf("Can't find region: %v", err)
```

Comment:
nit: fyi, you should wrap errors with `%w` rather than `%v`. `%w` provides more functionality and preserves stack traces better. people do it incorrectly all over our code base so i point it out when i see it: https://go.dev/blog/go1.13-errors

Reply:
Ahh, I see! Changed to %w